src/themes/mainTheme.ts

```diff
@@ -200,6 +200,15 @@ export const mainTheme = createTheme(
           },
         },
       },
+      MuiCssBaseline: {
+        styleOverrides: {
+          fieldset: {
+            border: 'none',
+            padding: 0,
+            margin: 0,
```

Comment:
ligger fortsatt mer padding på legend her. Merkes godt om man har en fane i dev og en lokalt og hopper frem og tilbake
```suggestion
            margin: 0,
            legend: {
              padding: 0,
            },
```

![bilde](https://github.com/user-attachments/assets/fcb5d62b-4bb2-47bd-a306-927bc7a90eef)